packages/mobile/src/components/share-drawer/ShareDrawer.tsx

```diff
@@ -291,8 +291,8 @@ export const ShareDrawer = () => {
         titleIcon={IconShare}
         styles={{ row: styles.row }}
       >
-        {content?.type !== 'playlist' || content?.playlist?.is_private ? (
-          <Text style={styles.titleHelperText} fontSize={'large'}>
+        {content?.type === 'playlist' && isOwner ? (
```

Comment:
wait actually i think this might only be meant for if the playlist is hidden so might want to keep that check in there @dylanjeffers

Reply:
oh yeah duh, thanks